index.html

```diff
@@ -115,10 +115,22 @@
     </style>
   </head>
   <body>
-    <script type="text/javascript" src="jssrc/commonmark.js"></script>
-    <script type="text/javascript" src="jssrc/codemirror.js"></script>
-    <script type="text/javascript" src="jssrc/microReact.js"></script>
-    <script type="text/javascript" src="jssrc/renderer.js"></script>
-    <script type="text/javascript" src="jssrc/editor.js"></script>
+    <script type="text/javascript" src="js/codemirror.js"></script>
+
+    <script type="text/javascript" src="js/system.js"></script>
+    <script>
+      SystemJS.config({
+        baseURL: "/js",
+        defaultJSExtensions: true,
+        meta: {
+          codemirror: {format: "amd"}
```

Comment:
I'm 95% sure I don't need this meta bit since I ended up registering it manually.